Sources/UINotificationPresentationContext.swift

```diff
@@ -73,8 +73,8 @@ public final class UINotificationPresentationContext {
         notificationView.topConstraint = notificationViewTopConstraint
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/UINotificationPresentationContext.swift;
  Line: 74;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)